File: fizzbuzz/fizzbuzz.js
  function fizzFind() {
    if (number % 3 == 0 && number % 5 == 0)
        return "fizzbuzz"
    else if (number % 3 == 0)
        return "fizz";
    else if (number % 5 == 0)
       return "buzz";
    else
       return number;
 }

  function fizzLoop(num) {
    number = 0;

    while (number <= num) {
       console.log(fizzFind(number));
       number++;
    }

  }

fizzLoop(50);

File: fizzbuzz/fizzbuzz.rb
class Fizzbuzz

  def initialize
    @number = 0
  end

  def run(number)
    if number % 3 == 0 && number % 5 == 0
       'fizzbuzz'
    elsif number % 3 == 0
       'fizz'
    elsif number % 5 == 0
       'buzz'
    else
       number
    end
  end

  def fizzLoop(num)
    while @number <= num
      puts self.run(@number)
      @number += 1
    end
  end

end

object = Fizzbuzz.new

object.fizzLoop(50)
